epyqlib/device.py

```diff
@@ -361,6 +363,7 @@ def _load_config(
             "access_password_path",
             "ParameterQuery;FactoryAccess;FactoryAccess",
         )
+        d.setdefault("software_hash_path", None)
```

Comment:
Not a fan of a variable named `d`, I understand it predates this ticket.

Reply:
Renamed in b988885fb3c9d284410ca9c1f83a0a902bb2aaeb